cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -242,8 +249,9 @@ define(['jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'common/j
                     'course-outline', 'xblock-string-field-editor', 'modal-button',
                     'basic-modal', 'course-outline-modal', 'release-date-editor',
                     'due-date-editor', 'grading-editor', 'publish-editor',
-                    'staff-lock-editor', 'content-visibility-editor', 'settings-modal-tabs',
-                    'timed-examination-preference-editor', 'access-editor', 'show-correctness-editor'
+                    'staff-lock-editor', 'unit-access-editor', 'content-visibility-editor',
```

Comment:
Out of curiosity, why is it necessary for this spec to know about all these templates, even though the spec does not seem to be testing the dialog? Not a big deal, just trying to understand.